iset.mm

```diff
@@ -67755,6 +67771,96 @@ elements or fails to hold (is equal to ` (/) ` ) for some element.
   $}
 
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Markov's principle
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c Markov $.  $( Class of all Markov sets. $)
+
+  $( Extend class definition to include the class of Markov sets. $)
+  cmarkov $a class Markov $.
+
+  ${
+    $d f x y $.
+    $( A Markov set is one where if a predicate (here represented by a function
+       ` f ` ) on that set does not hold (where hold means is equal to ` 1o ` )
+       for all elements, then there exists an element where it fails (is equal
+       to ` (/) ` ).  Generalization of definition 2.5 of [Pierik], p. 9.
+
+       In particular, ` _om e. Markov ` is known as Markov's Principle (MP).
+       (Contributed by Jim Kingdon, 18-Mar-2023.) $)
+    df-markov $a |- Markov = { y | A. f ( f : y --> 2o
+        -> ( -. A. x e. y ( f ` x ) = 1o -> E. x e. y ( f ` x ) = (/) ) ) } $.
+  $}
+
+  ${
+    $d A f x y $.
+    $( The predicate of being Markov.  (Contributed by Jim Kingdon,
+       18-Mar-2023.) $)
+    ismkv $p |- ( A e. V -> ( A e. Markov <-> A. f ( f : A --> 2o
+        -> ( -. A. x e. A ( f ` x ) = 1o
+          -> E. x e. A ( f ` x ) = (/) ) ) ) ) $=
+      ( vy cv c2o wf cfv c1o wceq wral wn c0 wrex wi wal cmarkov feq2 imbi12d
+      raleq notbid rexeq albidv df-markov elab2g ) EFZGCFZHZAFUHIZJKZAUGLZMZUJN
+      KZAUGOZPZPZCQBGUHHZUKABLZMZUNABOZPZPZCQEBRDUGBKZUQVCCVDUIURUPVBUGBGUHSVDU
+      MUTUOVAVDULUSUKAUGBUAUBUNAUGBUCTTUDAECUEUF $.
+  $}
+
+  ${
+    $d A f x $.  $d V f $.
+    $( The predicate of being Markov stated in terms of set exponentiation.
+       (Contributed by Jim Kingdon, 18-Mar-2023.) $)
+    ismkvmap $p |- ( A e. V -> ( A e. Markov <-> A. f e. ( 2o ^m A )
+        ( -. A. x e. A ( f ` x ) = 1o
+          -> E. x e. A ( f ` x ) = (/) ) ) ) $=
+      ( wcel cmarkov cv c2o cmap co cfv c1o wceq wral wn c0 wrex wi wal com wf
+      ismkv wb 2onn elmapg mpan imbi1d albidv bitr4d df-ral syl6bbr ) BDEZBFEZC
+      GZHBIJZEZAGUNKZLMABNOUQPMABQRZRZCSZURCUONULUMBHUNUAZURRZCSUTABCDUBULUSVBC
+      ULUPVAURHTEULUPVAUCUDHBUNTDUEUFUGUHUIURCUOUJUK $.
+  $}
+
+  ${
+    $d A f x $.
+    $( An omniscient set is Markov.  In particular, the case where ` A ` is
+       ` _om ` means that the Lesser Principle of Omniscience (LPO) implies
+       Markov's Princple (MP).  (Contributed by Jim Kingdon, 18-Mar-2023.) $)
+    omnimkv $p |- ( A e. Omni -> A e. Markov ) $=
```

Comment:
"principle" (twice)
and I would find `$p |- ( Omni C_ Markov ) $=` cleaner.

Reply:
> "principle" (twice)

Fixed, thanks.